src/main/java/edu/tcu/cs/superfrogscheduler/user/entity/SuperFrogUser.java

```diff
@@ -106,4 +122,26 @@ public PaymentPreference getPaymentPreference() {
     public void setPaymentPreference(PaymentPreference paymentPreference) {
         this.paymentPreference = paymentPreference;
     }
+
+    public PaymentForm generatePaymentForm(List<Request> requests, Period paymentPeriod) {
```

Comment:
I think generatePaymentForm() function should not be in the this entity as the entity should only serve as a design for user table, can you refactor it or move it to PaymentService?

Reply:
Its easier to keep it here and this is how he did it in his example so I'm sure he's fine with it